src/qualimap/qualimap_rnaseq/help.txt

```diff
@@ -0,0 +1,52 @@
+QualiMap v.2.3
```

Comment:
Since this component seems to be a port from nf-core, can you make sure that all arguments have been added to the component? Looks ok to me, but can you make sure?